### PR TITLE
Create user model

### PR DESCRIPTION
**Pull request by mattwalo32**

### CODE REVIEW

File: backend/api/models/Cars.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from mongoengine import *
+from flask_mongoengine import document
+from .base import db
+from api.core import Mixin
+
+class Cars(Document, Mixin):
```
Comment (shreyshrey1) [inline backend/api/models/Cars.py]: Should all these fields be true?

File: backend/api/models/Users.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+from mongoengine import *
+from flask_mongoengine import document
+from .base import db
+from api.core import Mixin
+
+class User(Document, Mixin):
+    user_id = IntField(required = True)
+    past_trips = IntField(required = True)
```
Comment (shreyshrey1) [inline backend/api/models/Users.py]: This should probably be a ```ListField``` of ```ObjectIdField```.

File: backend/api/models/Users.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
+from mongoengine import *
+from flask_mongoengine import document
+from .base import db
+from api.core import Mixin
+
+class User(Document, Mixin):
+    user_id = IntField(required = True)
```
Comment (shreyshrey1) [inline backend/api/models/Users.py]: Don't think you have explicitly declare the id field, mongodb will create an id automatically.

File: backend/api/models/Users.py (hunk 2 above)
Comment (shreyshrey1) [reply backend/api/models/Users.py]: Could use ```ReferenceField``` like you did for Cars too

Comment (shreyshrey1) [inline backend/api/models/Users.py]: is different from ```past_driver_trips``` and ```past_passenger_trips``` declared below?

File: backend/api/models/Users.py [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+class User(Document, Mixin):
+    user_id = IntField(required = True)
+    past_trips = IntField(required = True)
+    age = IntField(required = True)
+    rating = FloatField(required = True)
+    email = EmailField(required = True)
+    name = StringField(required = True)
+    phone = StringField(required = True)
+    venmo_handle = StringField(required = True)
```
Comment (shreyshrey1) [inline backend/api/models/Users.py]: consider making this not required?

File: backend/api/models/Cars.py (hunk 1 above)
Comment (abehara2) [reply backend/api/models/Cars.py]: I think past_trips doesn't necessarily have to be required

File: backend/api/models/Users.py [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+    
+    photo = StringField()
+    cars = ListField(ReferenceField(Cars))
+    past_driver_trips = ListField(ReferenceField(Trips))
+    past_passenger_trips = ListField(ReferenceField(Trips))
+    current_trips = ListField(ReferenceField(Trips))
+
```
Comment (uawasth2) [inline backend/api/models/Users.py]: can you add a __repr__ that is like the one in Person.py

File: backend/api/models/Cars.py [hunk 6]
```diff
@@ -0,0 +1,10 @@
+from mongoengine import *
+from flask_mongoengine import document
+from .base import db
+from api.core import Mixin
+
+class Cars(Document, Mixin):
+    year = IntField(required = True)
+    model = StringField(required = True)
+    color = StringField(required = True)
+    liscense_plate = StringField(required = True)
```
Comment (uawasth2) [inline backend/api/models/Cars.py]: Sanity check for license spelling.

**Pull request merged**